packages/graph/hash_graph/lib/graph/src/ontology/mod.rs

```diff
@@ -41,13 +41,13 @@ impl fmt::Display for AccountId {
 pub struct PersistedOntologyIdentifier {
     #[component(value_type = String)]
     uri: VersionedUri,
-    created_by: AccountId,
+    owned_by_id: AccountId,
```

Comment:
This let the tests on `packages/hash/integration` fail:
```
    ../api/src/model/ontology/data-type.model.ts:49:29 - error TS2339: Property 'createdBy' does not exist on type 'PersistedOntologyIdentifier'.

    49       accountId: identifier.createdBy,
                                   ~~~~~~~~~
```

Reply:
Thank you for catching (yet another reason for us to get on fixing CI soon 😁), it would seem some of my changes did not make it to being pushed